main.go

```diff
@@ -123,6 +129,10 @@ func main() {
 
 	watchNamespace, _ := os.LookupEnv(watchNamespaceEnvVar)
 	watchNamespaceSelector, _ := os.LookupEnv(watchNamespaceEnvSelector)
+	if watchNamespaceSelector != "" {
+		setupLog.Info(fmt.Sprintf("sharding is enabled via %s=%s. Beware: Always label Grafana CRs before enabling to ensure labels are inherited. Existing Secrets/ConfigMaps referenced in CRs also need to be labeled to continue working.", watchNamespaceEnvSelector, watchNamespaceSelector))
+	}
```

Comment:
Shouldn't the sharding log be dependent on `watchLabelSelectors`? Or am I misunderstanding something?

Reply:
You are correct I've push a commit to address that.